developer/wip/for-contributors/contributor-handbook.md

```diff
@@ -794,7 +802,196 @@ available. It uses the [Dataplane Signaling Protocol](#29-data-plane-signaling)
+
+This is sometimes referred to as the _provisioning phase_. If no additional provisioning is needed, the transfer process
+simply transitions through the state with a NOOP.
+
+Once that is done, the transfer begins in earnest. Data is transmitted according to the `dataDestination`, that was
+passed in the initiate-request.
+
+Once the transmission has completed, the transfer process will transition to the `COMPLETED` state, or - if an error
+occurred - to the `TERMINATED` state.
+
+The Management API provides several endpoints to manipulate data transfers.
+
+Here's a diagram of the state machine applied to transfer processes on consumer side:
+
+![Transfer Process Consumer State Machine](diagrams/transfer-process-consumer-states.png)
+
+Here's a diagram of the state machine applied to transfer processes on provider side:
+
+![Transfer Process Provider State Machine](diagrams/transfer-process-provider-states.png)
+
+A transfer process can be initiated from the consumer side by sending a `TransferRequest` to the connector management API:
+
+```json
+{
+    "@context": {
+        "@vocab": "https://w3id.org/edc/v0.0.1/ns/"
+    },
+    "@type": "https://w3id.org/edc/v0.0.1/ns/TransferRequest",
+    "protocol": "dataspace-protocol-http",
+    "counterPartyAddress": "http://provider-address",
+    "contractId": "contract-id",
+    "transferType": "transferType",
+    "dataDestination": {
+        "type": "data-destination-type"
+    },
+    "privateProperties": {
+        "private-key": "private-value"
+    },
+    "callbackAddresses": [
+        {
+            "transactional": false,
+            "uri": "http://callback/url",
+            "events": [
+                "contract.negotiation",
+                "transfer.process"
+            ],
+            "authKey": "auth-key",
+            "authCodeId": "auth-code-id"
+        }
+    ]
+}
+```
+
+where:
+
+- `counterPartyAddress`: the address where to send the `TransferRequestMessage` via the specified `protocol` (currently [`dataspace-protocol-http`](#27-protocol-extensions-dsp))
+-  `contractId`: the ID of a previously negotiated [contract agreement](#215-contract-agreements) through the [contract negotiation](#214-contract-negotiations) process.
+- [`transferType`](#2171-transfer-and-data-flows-types) and the [`dataDestination`](#2172-about-data-destinations) define how and where the data transfer should happen.
+- [`callbackAddresses`](#2173-transfer-process-callbacks) custom hooks in order bo be notified about state transition of the transfer process.
+- `privateProperties`: custom properties not shared with the counter party.
+
+##### 2.1.7.1 Transfer and data flows types
+
+The transfer type defines the channel (Distribution) for the data transfer and it depends on the capabilities of the [data plane](#3-the-data-plane). The `transferType` available for a data offering is available in the `dct:format` of the `Distribution` when inspecting the [catalog](#216-catalog) response.
```

Comment:
```suggestion
The transfer type defines the channel (Distribution) for the data transfer and it depends on the capabilities of the [data plane](#3-the-data-plane) if it can be fulfilled. The `transferType` available for a data offering is available in the `dct:format` of the `Distribution` when inspecting the [catalog](#216-catalog) response.
```